app/api/server/v1/inquiries.js

```diff
@@ -0,0 +1,37 @@
+import { Meteor } from 'meteor/meteor';
+
+import { API } from '../api';
+
+API.v1.addRoute('inquiries.get', { authRequired: true }, {
```

Comment:
Add `livechat` to your route:
```suggestion
API.v1.addRoute('livechat/inquiries', { authRequired: true }, {
```